### ISSUE TITLE
[a11y]: Page Header with breadcrumbs have violation due to same aria-label="Breadcrumb" on more than one <nav>

### ISSUE THREAD

**Issue by aknguyen7** [carbon-design-system/ibm-products#6445]
### Package

@carbon/ibm-products

### Browser

Chrome

### Package version

2.53

### React version

1.54

### Automated testing tool and ruleset

IBM Equal Accessibility Checker version 7.3

### Description

Using PageHeader with breadcrumb has accessibility violation due to the <nav> has the same aria-label="Breadcrumb". 

https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-page-header-pageheader--with-breadcrumbs&globals=viewport:basic

There is not an option to add/apply an aria-label for the breadcrumbs in PageHeader.

Slack: https://ibm-cloud.slack.com/archives/C013ZTX0N6B/p1731617428305279

### Reproduction/example

https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-page-header-pageheader--with-breadcrumbs&globals=viewport:basic

### Steps to reproduce

1. Create a PageHeader with breadcrumbs for example
<PageHeader
            breadcrumbs={[
                {
                href: '/integration',
                key: 'integrations',
                label: 'manage.integrations'
                },
            ]}
            title={{
                loading: false,
                text: 'cloud.monitoring.integrate'
            }}
            subtitle={'cloud.monitoring.subtitle'}
        />
2. Run IBM Accessbility Checker version 7.3

![Image](https://github.com/user-attachments/assets/0ba4db4f-f8a9-4407-b330-36493f565637)

![Image](https://github.com/user-attachments/assets/88408903-d4a8-4acf-80b4-84a150f21c0f)

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by amal-k-joy** [comment]
@aknguyen7 ,
Looks like an option to pass breadcrumb aria-label has been added recently as part of [this pr](https://github.com/carbon-design-system/ibm-products/pull/6398/files#diff-dfc4cc9013e246ca4df5738cb000e5c756a29218de33aa05d1de41b8291e2e46).

**Comment by steven1046** [comment]
@amal-k-joy do you know what version of Carbon it'll be included in?
